### ISSUE TITLE
Control Panel Translations

### ISSUE THREAD

**Issue by JohnathonKoster** [Stillat/meerkat-v1hub#7]
The following translation keys are incorrectly displayed within the comments section of the Control Panel:

* Addons.Meerkat::comments.comments
* Addons.Meerkat::actions.view

<img width="1766" alt="mkcptrans" src="https://user-images.githubusercontent.com/5232890/56869675-04b7ad00-69c9-11e9-8d8b-d51fb9a7998f.png">

**Comment by nightillusions** [comment]
![Also this ones do not work](https://user-images.githubusercontent.com/9394520/56877929-b6022580-6a51-11e9-890e-60b41f6cbcd9.jpg)

Statamic v2.11.12
Meerkat v1.5.4

**system.yaml**
```
locales:
  de:
    full: de_DE
    name: Deutsch
    url: /
  en:
    name: English
    full: en_US
    url: /en/
timezone: UTC
date_format: 'F jS, Y'
default_extension: md
filesystems:
  content:
    driver: local
    root: site/content
  storage:
    driver: local
    root: site/storage
  users:
    driver: local
    root: site/users
  themes:
    driver: local
    root: site/themes
    url: /site/themes
redactor:
  -
    name: Standard
    settings:
      buttons:
        - formatting
        - bold
        - italic
        - link
        - unorderedlist
        - orderedlist
        - html
  -
    name: Basic
    settings:
      buttons:
        - bold
        - italic
yaml_parser: symfony
send_powered_by_header: true
ensure_unique_ids: true
csrf_exclude: {  }
php_max_memory_limit: null
```
**meerkat.yaml**
```
auto_publish: true
auto_publish_authenticated_users: true
cp_avatar_driver: gravatar
automatically_close_comments: '0'
auto_check_spam: true
auto_delete_spam: false
auto_submit_results: false
form_user_fields:
  name: first_name
  email: email
akismet_fields:
  author: name
  email: email
  user_ip: user_ip
  user_agent: user_agent
  referrer: referrer
  permalink: page_url
  content: comment
iplist:
  allow:
    - 127.0.0.1
  block: ''
```

If you need more info, I could give you access to my website: https://jordin.eu or to the repo.

**Comment by nightillusions** [comment]
Okay, when I copied the `lang` folder `en` and renamed it to my language string `de` it works. But I think it should fallback to `en` when there is no translation present.

**Comment by JohnathonKoster** [comment]
Interesting, I'll look into this behavior. Thanks for the additional information, it really does help :)

**Comment by JohnathonKoster** [comment]
Digging through this some more, it also seems that the Statamic CP itself would suffer this same issue (Meerkat uses Statamic's translate helper methods). I am investigating some solutions for this, but they would be Meerkat specific, and hope to generalize them enough to be used by Statamic.

**Comment by JohnathonKoster** [comment]
Fixed in v1.5.62
